bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -542,6 +542,22 @@
 		<config-description-ref uri="thing-type:boschshc:device"/>
 	</thing-type>
 
+
+	<thing-type id="presence-simulation">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="shc"/>
+		</supported-bridge-type-refs>
+
+		<label>Presence Simulation</label>
+		<description>Simulates that someone is home.</description>
+		<semantic-equipment-tag>Application</semantic-equipment-tag>
```

Comment:
During review and tests I saw, that you used here "Application" as the equipment-tag.
The `<thing-type id="user-defined-state">` doesn't have any equipment-tag yet. Shall we use "Application" to for the user defined state?

Reply:
I think the `application` can be applied here. @andrewfg is the export on this!